src/api/loaders/csv.js

```diff
@@ -0,0 +1,9 @@
+import ezs from 'ezs';
+import ezsBasics from 'ezs-basics';
+
+ezs.use(ezsBasics);
+
+export default config => stream =>
+  stream
```

Comment:
ok, but where is the specific config file, I don't find any file like this https://github.com/Inist-CNRS/lodex-v1/blob/master/.eslintrc.js